File: youtubejextractor/src/main/java/com/github/kotvertolet/youtubejextractor/models/test/SignalServiceEndpoint.java
package com.github.kotvertolet.youtubejextractor.models.test;

import java.util.List;
import com.google.gson.annotations.SerializedName;

public class SignalServiceEndpoint{

	@SerializedName("signal")
	private String signal;

	@SerializedName("actions")
	private List<ActionsItem> actions;

	public void setSignal(String signal){
		this.signal = signal;
	}

	public String getSignal(){
		return signal;
	}

	public void setActions(List<ActionsItem> actions){
		this.actions = actions;
	}

	public List<ActionsItem> getActions(){
		return actions;
	}
}

File: youtubejextractor/src/main/java/com/github/kotvertolet/youtubejextractor/models/newModels/SignalServiceEndpoint.kt
package com.github.kotvertolet.youtubejextractor.models.newModels

import com.google.gson.annotations.SerializedName

class SignalServiceEndpoint {
    @SerializedName("signal")
    var signal: String? = null

    @SerializedName("actions")
    var actions: List<ActionsItem>? = null
    override fun toString(): String {
        return "SignalServiceEndpoint{" +
                "signal = '" + signal + '\'' +
                ",actions = '" + actions + '\'' +
                "}"
    }
}